### PR TITLE
Add  profile and templates for the dockerized discovery task

### PR DESCRIPTION
**Pull request by keedya**
depends on : https://github.com/RackHD/on-taskgraph/pull/149
depends on: https://github.com/RackHD/on-imagebuilder/pull/52/files
depends on : https://github.com/RackHD/on-tasks/pull/293

references:  
- https://osclouds.wordpress.com/2015/11/23/announcing-a-new-docker-based-hanlon-microkernel
- https://groups.google.com/forum/#!topic/rackhd/cSYYxhtPWY0

### CODE REVIEW

**Comment by keedya** [comment]
@RackHD/corecommitters @zyoung51 @jlongever @benbp

**Comment by yyscamper** [comment]
:+1:

**Comment by benbp** [comment]
👍

File: data/templates/cloud-config.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#cloud-config
+write_files:
+  - path: /etc/rc.local
+    permissions: "0755"
+    owner: root
+    content: |
+      #!/bin/bash
+      modprobe ipmi_devintf
+      wget -O /tmp/micro.tar.xz <%= api.server %>/micro.tar.xz
```
Comment (benbp) [inline data/templates/cloud-config.yaml]: More descriptive filename here too? Also, should we store it in /common?

Comment (keedya) [inline data/templates/cloud-config.yaml]: Any suggestions? @benbp

Comment (anhou) [inline data/templates/cloud-config.yaml]: does it mean microkenrel? if yes, how about 'microkernel', if not, maybe it could be micore<what>

Comment (zyoung51) [inline data/templates/cloud-config.yaml]: This is the discovery image as a docker container.  We should call it discovery.docker.tar.xz.  Agree with @benbp that it should be moved to /common.

**Comment by keedya** [comment]
@anhou @zyoung51 @benbp all set!!

**Comment by brianparry** [comment]
test this plesae

**Comment by keedya** [comment]
test this plesae

**Comment by brianparry** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Pull request merged**